extensions/amp-a4a/0.1/amp-ad-utils.js

```diff
@@ -78,13 +78,21 @@ export function getAmpAdMetadata(creative) {
   try {
     const metaDataObj = parseJson(
         creative.slice(metadataStart + metadataString.length, metadataEnd));
-    const ampRuntimeUtf16CharOffsets =
+    let ampRuntimeUtf16CharOffsets =
       metaDataObj['ampRuntimeUtf16CharOffsets'];
     if (!isArray(ampRuntimeUtf16CharOffsets) ||
```

Comment:
Can we limit this change to just when ampRuntimeUtf16CharOffsets does not exist as opposed to also when its malformed?

Reply:
Done.